PerfectDay/Table_schedule.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System;
+
+namespace PerfectDay
+{
+    public class Schedule
+    {
+        public int Schedule_Id { get; set; }
```

Comment:
Please set the name ScheduleId. (https://www.entityframeworktutorial.net/code-first/code-first-conventions.aspx)